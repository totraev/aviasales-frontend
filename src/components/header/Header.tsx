import React, { SFC } from "react";
import styles from "./Header.module.css";

const Header: SFC = () => (
  <div className={styles.header}>
    <img src={require("./svg/logo.svg")} alt="Aviasales" />
  </div>
);

export default Header;
