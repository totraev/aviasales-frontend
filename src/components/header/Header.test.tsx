import React from 'react'
import renderer from 'react-test-renderer'

import Header from './Header'

describe('Header component', () => {
  it('Should renders correctly', () => {
    const tree = renderer.create(<Header />).toJSON()

    expect(tree).toMatchSnapshot()
  })
})
