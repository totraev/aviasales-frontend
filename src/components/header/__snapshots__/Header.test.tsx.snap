// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component Should renders correctly 1`] = `
<div
  className="header"
>
  <img
    alt="Aviasales"
    src={
      Object {
        "ReactComponent": [Function],
        "__esModule": true,
        "default": "logo.svg",
      }
    }
  />
</div>
`;
